src/gui/accountstate.h

```diff
@@ -201,7 +201,7 @@ public slots:
     void stateChanged(State state);
     void isConnectedChanged();
     void hasFetchedNavigationApps();
-    void userStatusChanged();
+    void statusChanged();
```

Comment:
@camilasan 

For the record. I did not mind having `userStatusChanged` here, as the class name is `AccountState`, so, it's fine to have `AccountState::userStatusChanged`.

My comment was for that one in the **UserModel.h** file :)

Reply:
sorry, I will leave it like that. I just wanted to be consistent.